test/t8_test_cmesh_partition.cxx

```diff
@@ -24,6 +24,7 @@
 #include <t8_schemes/t8_default_cxx.hxx>
 #include "t8_cmesh/t8_cmesh_trees.h"
 #include "t8_cmesh/t8_cmesh_partition.h"
+#include "src/t8_refcount.h"
```

Comment:
```suggestion
```

Do we need t8_refcount here? We do not use any functions of t8_refcount in this test

Reply:
I asked @holke the same thing in a private message, as this line also lead to a merge conflict with the line below. From my point, we could delete this line.
His answer was to include it in this merge, as it could lead to other merge conflicts in the future. Maybe we can discuss that together tomorrow.